internal/cli/up.go

```diff
@@ -211,6 +214,9 @@ func provideWebPort() model.WebPort {
 	return model.WebPort(webPort)
 }
 
+func provideNoBrowserFlag() bool {
```

Comment:
can you create a `type NoBrowser bool` for this?

it's not just any boolean, it's a boolean for a specific thing

Reply:
☝️ to add context to this: remember when we were looking at the generated wire code and i was so surprised that it put the boolean in the right place? that's because wire is basically guessing what function output to pass to what function, based on type. there's probably some magic that makes it more accurate but it's still kind of a guess that THIS boolean gets passed to THAT function.

by making a type alias, it's just a boolean under the hood, but wire knows that this function returns a `NoBrowser` and this other function accepts a `NoBrowser`, and they're the only two functions that do that, so it's really clear what code to generate. Does that make sense?

Reply:
ooo ok noted! Thank you